assets/src/dashboard/app/views/templates/detail.js

```diff
@@ -129,6 +160,16 @@ function TemplateDetail() {
                     <ColorList colors={template.colors} size={30} />
                   </MetadataContainer>
                 </DetailContainer>
+                <NavButton
+                  aria-label={__('View next template', 'web-stories')}
+                  onClick={handleNextTemplateClick}
+                  disabled={
+                    !templates?.length ||
+                    activeTemplateIndex === templates.length - 1
+                  }
+                >
+                  <RightArrow {...ICON_METRICS.LEFT_RIGHT_ARROW} />
```

Comment:
Even though RTL support is not currently a priority, it would be rather easy to switch the arrows for RTL screens. Something like this:

```js
const { isRTL } = useConfig();

const PrevArrow = isRTL ? RightArrow : LeftArrow;
const NextArrow = isRTL ? LeftArrow : RightArrow;

<NavButton>
  <PrevArrow />
</NavButton>
```

Using "Switch to RTL" at the top you can easily test the RTL layout

Reply:
Didn't even think about this, nice call out, I'll update.